openmetadata-ui/src/main/resources/ui/src/pages/AddQueryPage/AddQueryPage.component.tsx

```diff
@@ -88,6 +88,19 @@ const AddQueryPage = () => {
     }
   };
 
+  const getLabelValue = (
+    displayName: string,
+    name: string,
+    fullyQualifiedName: string
+  ): JSX.Element => (
+    <Space direction="vertical" size={0}>
+      <Typography.Text>{getEntityName({ displayName, name })}</Typography.Text>
+      <Typography.Text className="text-gray-400 text-xs break-word">
+        {fullyQualifiedName}
+      </Typography.Text>
+    </Space>
+  );
```

Comment:
we can move this to utils files, and use with import!

Reply:
Moved this function to `EntityUtils.tsx`